### PR TITLE
Upgrade blob lib part 7.1 - refactor func test

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

Upgrade blob lib part func test.
move conf to one class so they will not reload per test class.
remove lease usage in sas test as it is not necessary. 
use var.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java [hunk 1]
```diff
@@ -22,58 +21,42 @@
 
 import static com.jayway.awaitility.Awaitility.await;
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.bulkscanprocessor.config.Configs.STORAGE_ACCOUNT_KEY;
+import static uk.gov.hmcts.reform.bulkscanprocessor.config.Configs.STORAGE_ACCOUNT_NAME;
+import static uk.gov.hmcts.reform.bulkscanprocessor.config.Configs.STORAGE_ACCOUNT_URL;
+import static uk.gov.hmcts.reform.bulkscanprocessor.config.Configs.*;
```
Comment (doncem) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java:27]: we are using explicit imports ;)

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java [hunk 2]
```diff
@@ -9,9 +9,8 @@
 import com.azure.storage.blob.BlobServiceClientBuilder;
 import com.azure.storage.blob.specialized.BlobLeaseClientBuilder;
 import com.azure.storage.common.StorageSharedKeyCredential;
-import com.typesafe.config.Config;
-import com.typesafe.config.ConfigFactory;
 import uk.gov.hmcts.reform.bulkscanprocessor.TestHelper;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.Configs;
```
Comment (doncem) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java:13]: If all are imported statically, perhaps this one is not needed?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/config/Configs.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.config;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class Configs {
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/config/Configs.java:6]: can we plz rename this class to `TestConfiguration` ?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java [hunk 4]
```diff
@@ -84,7 +74,7 @@ public void setUp() throws Exception {
 
         BlobServiceClient blobServiceClient = blobServiceClientBuilder.buildClient();
 
-        String inputContainerName = config.getString("test-storage-container-name");
+        String inputContainerName = STORAGE_CONTAINER_NAME;
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java:77]: do we need this `inputContainerName` variable  can directly use `STORAGE_CONTAINER_NAME` ?

Comment (mdayican) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java:77]: I think this is done for readability of the code.

**Pull request merged**